target/linux/ramips/dts/mt7620a_dlink_dap-1620-a2.dts

```diff
@@ -0,0 +1,203 @@
+#include "mt7620a.dtsi"
```

Comment:
Please insert `// SPDX-License-Identifier: ` with identifier(s) of your preferred license.

https://spdx.org/licenses/